LoggingKata.Test/TacoParserTests.cs

```diff
@@ -12,18 +12,27 @@ public void ShouldDoSomething()
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("0, 0, test")]

```

Comment:
Consider adding some more InlineData to test different cases, such as "20.5, -20.5, Test", "-20.5, 20.5, Test", etc.